src/employees/utils/imageHandler/local.ts

```diff
@@ -45,9 +45,9 @@ const toFileName = (name: string): string => {
 const downloadAndStore: (
   path: string,
   dir: string,
-  image: any,
-) => Promise<void> = async (fileName: string, dirPath: string, image: any) => {
-  const request = await fetch(image.large.url);
+  image: ApiEmployee['image'],
+) => Promise<void> = async (fileName, dirPath, image) => {
+  const request = await fetch(image.fit_thumb.url);
```

Comment:
Are we concerned about image size on backend, won't sizing be auto solved with the `next/Image` component?

Reply:
Hmm. Dete er mulig vi kan ha de større nå og la caches. Må undersøke litt mer om hvordan det faktisk blir med next/Image for det ser ut som det kanskje blir resiza on load (og kanskje da cacha?). Ikke satt meg inn i hvordan det faktisk gjøres. Jeg gikk litt frem og tilbake mellom next/Image og ikke her så endra på et tidspunkt da, men burde kanskje være large ja.